src/components/body/audio/time.js

```diff
@@ -0,0 +1,29 @@
+import { TimeContext } from './sound-context'
+import React from 'react'
+function getMinSecStr(secs) {
+  const minutes = Math.floor(secs / 60)
+  const seconds = Math.floor(secs - minutes * 60)
+  return `${minutes}:${
+    seconds < 10 ? 0 + seconds.toString() : seconds.toString()
```

Comment:
> Question:
> Do we need `React.Context` here?
> It seems that we can pass these props into each children(`SeekBar` and `ControlButton`) directly.

The pros of using context is that we can pack the data and methods that  are related to audio all into context. It make `index.js` clean, just contain the layout of view. And we can easily reuse the audio context in the future with different layout. 

The cons I think is that if multiple contexts are used in a single component we need to nest the consumer to retrieve the data.

Reply:
I thought `React.Context` is used to avoid too many props from passing through lots of components. 
Ref: https://reactjs.org/docs/context.html#when-to-use-context

Current solution is pretty good, however, I just think if we can use `props`, then we use `props` directly.